server/src/routines/game.ts

```diff
@@ -714,10 +714,9 @@ function* checkDeckedOut(game: GameModel) {
 		game.settings.unlimitedCards
 	)
 		return []
-	return [game.currentPlayer, game.opponentPlayer].flatMap((player) => {
-		if (player.deckedOut) return [player]
-		return []
-	})
+	return [game.currentPlayer, game.opponentPlayer].filter(
+		(player) => player.deckedOut,
+	)
```

Comment:
I would prefer if you didn't touch this here since screaper is working on similar things ATM it might cause a conflict

Reply:
Kinda confused why this changed because I didn't touch it.